### DISCUSSION TITLE
setBotMeta

### DISCUSSION THREAD

**Discussion by dodujy27** [storycraft/node-kakao#707] [Q&A]
openlink에  setBotMeta는 어디에 사용 되고 어떻게 쓰는 건가요?
![BDC35334-66E9-47FA-879C-C2E2DFA23382](https://user-images.githubusercontent.com/60725138/135741299-02d29770-c71a-4669-ae2b-d9190da7f89b.jpeg)
![C91D195C-A70E-4FC1-BBC7-C71805624F0B](https://user-images.githubusercontent.com/60725138/135741303-7b200b1f-7889-409e-ac07-3f794841079a.jpeg)

그리고 BotMetaContent에 무엇을 넣는지도 궁금합니다.

**Comment by artist012** [comment]
봇메타는 막힌걸로 압니다ㅡㅡ
